src/main/kotlin/ar/edu/unahur/obj2/tareas/Tarea.kt

```diff
@@ -1,3 +1,67 @@
 package ar.edu.unahur.obj2.tareas
 
-class Tarea
+interface ITarea {
```

Comment:
Ok, aunque la notación de `I` inicial para interfaces es una convención muy difundida en C# pero que no tiene muchos argumentos que la sostengan. Si les interesa, les dejo un debate (en inglés) al respecto: [Should interface names begin with an “I” prefix?
](https://softwareengineering.stackexchange.com/questions/117348/should-interface-names-begin-with-an-i-prefix)